<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu de aquisição</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="db_label">
       <property name="text">
        <string>Banco :</string>
       </property>
      </widget>
      <widget class="QComboBox" name="db_options">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>246</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>246</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="workspace_label">
       <property name="text">
        <string>Área de Trabalho :</string>
       </property>
      </widget>
      <widget class="QComboBox" name="workspace_options">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>246</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>246</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="profile_label">
       <property name="text">
        <string>Perfil :</string>
       </property>
      </widget>
      <widget class="QComboBox" name="menu_options">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>246</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>246</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="menu_reclassify">
      <property name="text">
       <string>Ativar reclassificação</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QTabWidget" name="menu_area_buttons">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Pesquisar :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="menu_search"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
